writeengine/bulk/we_bulkload.cpp

```diff
@@ -153,6 +153,7 @@ BulkLoad::BulkLoad()
  , fEnclosedByChar('\0')
  ,  // not enabled unless user overrides enclosed by char
```

Comment:
Nit: I'd move all the constant initialization to members declaration using C++11 syntax, leaving default c-tor